torchci/lib/bot/drciBot.ts

```diff
@@ -23,8 +24,12 @@ async function getDrciComment(
   return {id: 0, body: ""};
 }
 
-export function formDrciComment(): string {
-  let body = "hello there!"
+export function formDrciComment(prNum: number): string {
+  const hudUrl = "hud.pytorch.org/pr/" + prNum;
```

Comment:
it's usually better to do something like 
` `hud.pytorch.org/pr/${prNum}` ` for better readability when string formatting. ( same for the other stuff on line 30 and 31).

Reply:
also maybe move this constant up.